cli/azd/pkg/input/asker.go

```diff
@@ -0,0 +1,179 @@
+package input
```

Comment:
nit.
Consider having one single PR to move the Asker from `cmd` to this new package.
IMOH, that change deserves its own commit

Reply:
The goal is to have it in one place and didn't want to introduce the refactoring to existing code till the next PR where we incorporate these changes back into the azd commands.